lms/envs/common.py

```diff
@@ -2088,3 +2088,19 @@
         'profile_image',
     ],
 }
+
+# PROFILE IMAGE CONFIG
+# TODO: add these settings to aws.py as well
+PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
+# PROFILE_IMAGE_DOMAIN points to the domain from which we serve image
+# files from.  When this is '/', it refers to the same domain as the
+# app server.  If serving from a different domain, specify that here
+# i.e. 'http://www.example-image-server.com/'
+PROFILE_IMAGE_DOMAIN = '/'
+PROFILE_IMAGE_URL_PATH = 'media/profile_images/'
+PROFILE_IMAGE_DEFAULT_FILENAME = 'default_profile_image'  # TODO: determine final name
```

Comment:
I agree with not doing fancy file installation stuff, especialy since that seems to cross into theming territory. Since I also don't think we should provide a default image to the open edx community, I'll advocate for just having a comment here.